### ISSUE TITLE
Unusual BLS change error

### ISSUE THREAD

**Issue by jrausch12** [ethereum/staking-deposit-cli#350]
I'm testing various scenarios and came across this:

`The given withdrawal credentials is matching the old BLS withdrawal credentials that mnemonic generated` 

I generated this error by giving the tool a validator index and its corresponding BLS withdrawal list, but purposely giving it another validator's mnemonic. 

So what does this error mean exactly? I'm trying to understand it so I can help others who encounter it.

Using binary on Ubuntu from PR-313

**Comment by wackerow** [comment]
@jrausch12 Thanks for posting this... As mentioned in Discord, would suggest giving the latest release a try: https://github.com/ethereum/staking-deposit-cli/releases/tag/v2.5.0

Personally getting the following error when the credentials don't match:

```log
[Error] The given withdrawal credentials does not match the old BLS withdrawal credentials that mnemonic generated.
```

Let me know if you're still getting this on `v2.5.0`, thanks!

**Comment by jrausch12** [comment]
@wackerow Resolved on v2.5.0, thanks! Someone should update this notes page to point to the releases page. https://notes.ethereum.org/@launchpad/btec#Option-1-binary-files
